CONTRIBUTING.md

```diff
@@ -99,13 +99,11 @@ Output format of any non-plumbing command can change in non-backward-compatible
    a [git-machete package](https://aur.archlinux.org/packages/git-machete) is hosted in Arch User Repository (AUR).
    If the release introduces significant changes/critical bugfixes, please [flag the package as out of date](https://aur.archlinux.org/pkgbase/git-machete/flag).
 
-<!--
 1. Verify that changes you made in files holding blogs content are reflected in the corresponding medium articles. Files and corresponding to them articles:
    * [blog-1.md](https://github.com/VirtusLab/git-machete/blob/develop/blogs/git-machete-1/blog-1.md) - [Make your way through the git (rebase) jungle with Git Machete](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02);
```

Comment:
Nit: rename the \*.md files for consistency maybe? since it's `blog-1`/`blog-2` here but just `blog` for Docker